skore-ui/src/views/ComponentsView.vue

```diff
@@ -15,7 +15,7 @@ import DraggableList from "@/components/DraggableList.vue";
 import DropdownButton from "@/components/DropdownButton.vue";
 import DropdownButtonItem from "@/components/DropdownButtonItem.vue";
 import DynamicContentRasterizer from "@/components/DynamicContentRasterizer.vue";
-import { type EditableListItemModel } from "@/components/EditableList.vue";
+import EditableList, { type EditableListItemModel } from "@/components/EditableList.vue";
```

Comment:
is this change here for a reason? otherwise, isn't the linter supposed to catch it?

Reply:
For some reason the component page was broken because the import was missing. Sadly the linter does not catch it.